apps/web/src/features/athletes/components/detail-sections/athlete-comms-discord.tsx

```
@@ -0,0 +1,55 @@
+		<Card>
+			<CardHeader>
+				<CardTitle className="flex items-center gap-2">
+					<MessageSquare className="h-5 w-5" />
+					Comms & Discord
+				</CardTitle>
+			</CardHeader>
+			<CardContent className="space-y-4">
+				<div>
+					<label className="font-medium text-muted-foreground text-sm">
+						Assigned Sending Email
+					</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)